scripts/common_ci.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3 -i
```

Comment:
I pulled over lots of python changes to make the safe struct generators and clang-format work. It is ugly copy copy though.